.github/workflows/aiexplainer-docker-publish.yml

```diff
@@ -1,4 +1,4 @@
-name: AIXExapliner Docker Publisher
+name: AIXExplainer Docker Publisher
```

Comment:
Hah, may as well fix the typo in `artexplainer-docker-publish.yml` as well?

Reply:
@pvaneck fixed!